front-end/src/components/PostListItem.tsx

```diff
@@ -90,6 +125,9 @@ export default function PostListItem(props: Props) {
           {EditCardLink()}
           {DeleteCardLink()}
           {LikeCardLink()}
+          {props.isReshareable ? (ReshareCardLink()): null}
```

Comment:
I think you can move the condition to the ReshareCardLink function and it should still work, just to keep it consistent with how these other ones are doing it.